sos/report/plugins/aap_controller.py

```diff
@@ -80,7 +78,7 @@ def setup(self):
 
     def postproc(self):
         # remove database password
-        jreg = r"(\s*\'PASSWORD\'\s*:(\s))(?:\"){1,}(.+)(?:\"){1,}"
+        jreg = r"(\s*'PASSWORD'\s*:\s*)('.*')"
```

Comment:
The password value is always in single quote marks? Could you show some example (ideally that failed to catch now and will be obfuscated by your RE), please?